docs/csharp/language-reference/keywords/formatting-numeric-results-table.md

```diff
@@ -1,32 +1,55 @@
-  
- For more information about standard and custom formatting strings, see [Formatting Types](../../../standard/base-types/formatting-types.md).
-  
-|Format Specifier|Description|Examples|Output|  
+# Formatting numeric results table (C# Reference)
+
+The following table shows supported format specifiers for formatting numeric results.
+
+|Format specifier|Description|Examples|Output|  
 |----------------------|-----------------|--------------|------------|  
-|C or c|Currency|Console.Write("{0:C}", 2.5);<br /><br /> Console.Write("{0:C}", -2.5);|$2.50<br /><br /> ($2.50)|  
-|D or d|Decimal|Console.Write("{0:D5}", 25);|00025|  
-|E or e|Scientific|Console.Write("{0:E}", 250000);|2.500000E+005|  
-|F or f|Fixed-point|Console.Write("{0:F2}", 25);<br /><br /> Console.Write("{0:F0}", 25);|25.00<br /><br /> 25|  
-|G or g|General|Console.Write("{0:G}", 2.5);|2.5|  
-|N or n|Number|Console.Write("{0:N}", 2500000);|2,500,000.00|  
-|X or x|Hexadecimal|Console.Write("{0:X}", 250);<br /><br /> Console.Write("{0:X}", 0xffff);|FA<br /><br /> FFFF|  
-  
-## See Also
-
-- [C# Reference](../../../csharp/language-reference/index.md)  
-- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
-- [Standard Numeric Format Strings](../../../standard/base-types/standard-numeric-format-strings.md)  
-- [Reference Tables for Types](../../../csharp/language-reference/keywords/reference-tables-for-types.md)  
-- [string](../../../csharp/language-reference/keywords/string.md)
+|C or c|Currency|`Console.Write("{0:C}", 2.5);`<br /><br /> `Console.Write("{0:C}", -2.5);`|$2.50<br /><br /> ($2.50)|  
+|D or d|Decimal|`Console.Write("{0:D5}", 25);`|00025|  
+|E or e|Exponential|`Console.Write("{0:E2}", 250000);`|2.50E+005|  
+|F or f|Fixed-point|`Console.Write("{0:F2}", 2.5);`<br /><br /> `Console.Write("{0:F0}", 2.5);`|2.50<br /><br /> 3|  
+|G or g|General|`Console.Write("{0:G}", 2.5);`|2.5|  
+|N or n|Numeric|`Console.Write("{0:N}", 2500000);`|2,500,000.00|  
+|P or p|Percent|`Console.Write("{0:P}", 0.25);`|25.00%|  
+|R or r|Round-trip|`Console.Write("{0:R}", 2.5);`|2.5|  
+|X or x|Hexadecimal|`Console.Write("{0:X}", 250);`<br /><br /> `Console.Write("{0:X}", 0xffff);`|FA<br /><br /> FFFF|  
+
+## Remarks
+
+The output presented in the table is generated with the "en-US" <xref:System.Globalization.CultureInfo>.
```

Comment:
I think that I would move this up to follow "The following table..." sentence. Placing it first makes it seem critically important and everything that follows less important. But putting it last in the section allows it to go completely unnoticed.